crates/ruff_python_formatter/src/statement/stmt_match.rs

```diff
@@ -36,7 +36,7 @@ impl FormatNodeRule<StmtMatch> for FormatStmtMatch {
         )
         .fmt(f)?;
 
-        let mut cases_iter = cases.iter();
+        let mut cases_iter = cases.iter().peekable();
```

Comment:
I'm probably blind but what do we need the `peekable` for?

Reply:
Good catch, not needed anymore